src/WebJobs.Script/ManagedDependencies/IManagedDependencyConfigs.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Script.Grpc.Messages;
+
+namespace Microsoft.Azure.WebJobs.Script.ManagedDependencies
+{
+    internal interface IManagedDependencyConfigs
+    {
+        string RapidUpdateAzModulePath { get; }
+
+        string ManagedDependencyHomeDirectoryWin { get; }
```

Comment:
Can we merge this and the following property? This should just be correctly populated based on the environment, that will avoid the additional complexity of checking the environment in order to use the right property everywhere this is used.